fleet-cmdr/src/handlers/cmd_orb_details.rs

```diff
@@ -0,0 +1,49 @@
+use async_trait::async_trait;
+use color_eyre::eyre::Result;
+use orb_relay_client::{QoS, RecvMessage};
+use orb_relay_messages::{
+    orb_commands::v1::{OrbCommandError, OrbDetailsRequest, OrbDetailsResponse},
+    prost::Message,
+};
+use tracing::info;
+
+use super::OrbCommandHandler;
+
+#[derive(Debug)]
+pub struct OrbDetailsCommandHandler {}
+
+impl OrbDetailsCommandHandler {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+#[async_trait]
+impl OrbCommandHandler for OrbDetailsCommandHandler {
+    #[tracing::instrument]
+    async fn handle(&self, command: &RecvMessage) -> Result<(), OrbCommandError> {
+        info!("Handling orb details command");
+        let _request = OrbDetailsRequest::decode(command.payload.as_slice()).unwrap();
+        // TODO(paulquinn00): Consult with @oldgalileo and @sfikastheo to determine where to get this info from.
+        let response = OrbDetailsResponse {
+            orb_id: "".to_string(),
+            orb_name: "".to_string(),
+            jabil_id: "".to_string(),
+            hardware_version: "".to_string(),
```

Comment:
Get this information from the MCUs or over DBus

Reply:
can't we just use mcu-util?

Reply:
Yes, that's fine too. @TheButlah we should probably honestly normalize the collection of all of this early-boot.